src/AutoRest.CSharp/LowLevel/Output/DpgClient.cs

```diff
@@ -21,7 +22,7 @@
 
 namespace AutoRest.CSharp.Output.Models
 {
-    internal class LowLevelClient : TypeProvider
+    internal class DpgClient : TypeProvider
```

Comment:
feels like the renaming can be a separate PR

Reply:
sure